10up/classifai
# ClassifAI NUX

### Description of the Change
PR Adds new user onboarding and updates existing settings UI.

Closes #30 

#### Screenshots:
**Activation Notice:**
<img width="1339" alt="image" src="https://user-images.githubusercontent.com/10613171/229032892-4eaca467-f437-4275-8b1b-963e5a8dfabd.png">

**ClassifAI Step 1:** 
![screencapture-classifai-local-wp-admin-admin-php-2023-04-28-20_06_33](https://user-images.githubusercontent.com/10613171/235177236-4619bd3a-6fd6-4626-998b-0f0505271d2d.png)


**ClassifAI Step 2:** 
![Screenshot 2023-03-31 at 11 16 01 AM](https://user-images.githubusercontent.com/10613171/229034308-c820255a-20d3-4085-9271-60121a5b65ff.png)

**ClassifAI Step 3:** 
![Screenshot 2023-03-31 at 11 14 31 AM](https://user-images.githubusercontent.com/10613171/229034289-f2b22b6b-e334-48c2-86ce-796d8a3ec441.png)

**Setup Summary:** 
<img width="1508" alt="Screenshot 2023-04-28 at 8 04 12 PM" src="https://user-images.githubusercontent.com/10613171/235177332-d02edfc6-a657-4bc5-abdb-ed022503c9c8.png">


**Classifai Settings:**
![Screenshot 2023-03-31 at 11 18 55 AM](https://user-images.githubusercontent.com/10613171/229034300-1cec0888-b439-40ce-948d-34d1ab49af0a.png)
![Screenshot 2023-03-31 at 11 19 05 AM](https://user-images.githubusercontent.com/10613171/229034294-7249fe96-9624-4194-a115-65153481ca8a.png)

### Reviewer Notes:
- Placeholder images are added in step 1 and the summary step. it will be replaced with actual images once we have images available. I have made PR ready for review to unblock the process here. (@jeffpaul could please help to get images for the same)
- E2E tests are not added as part of this PR, I will add it in the follow-up PR.

### How to test the Change
1. Install and activate the plugin.
2. Verify that the notice for start setup is there.
3. Start ClassifAI set up by clicking the button in the activation notice.
4. Go through each step of setup and make sure everything works fine
5. Go to ClassifAI settings and check the new UI and make sure everything works fine.

### Changelog Entry
> Added - Classifai Setup onboarding and UI improvements.

### Credits

Props @ryanwelcher @jeffpaul @iamdharmesh 


### Checklist:
